airstack-modules/graphql/airstack-domain-name-schema.graphql.ts

```diff
@@ -71,6 +72,7 @@ type AirDomainTransferTransaction implements AirDomainEvent @entity {
   tokenId: String # dec(labelhash)  # - NA
   domain: AirDomain!
   index: BigInt! # - NA
+  lastUpdatedIndex: BigInt!
```

Comment:
This field is only required in entity which will be updated. So it's need for Domain Entity only. Please remove it. 

Also refer lens schema for details